.env

```diff
@@ -0,0 +1,4 @@
+POSTGRES_USER=hello_django
+POSTGRES_PASSWORD=hello_django
+POSTGRES_DB=hello_django_dev
+PGDATA=/var/lib/postgresql/data
```

Comment:
You should not add the `.env` file with real data. 
You should add `env.example`, which will contain fake demo data instead of real variables.